components/JournalyEditor/JournalyEditor.tsx

```diff
@@ -1,30 +1,17 @@
 import React, { useEffect, useMemo, useCallback } from 'react'
 import { createEditor, Editor, Node } from 'slate'
-import {
-  Slate,
-  Editable,
-  withReact,
-  useSlate,
-  RenderElementProps,
-  RenderLeafProps,
-  ReactEditor,
-} from 'slate-react'
+import { Slate, withReact, ReactEditor } from 'slate-react'
 import { withHistory } from 'slate-history'
+import { pipe, TablePlugin, EditablePlugins } from '@udecode/slate-plugins'
 import isHotkey from 'is-hotkey'
-import classNames from 'classnames'
+
 import theme from '@/theme'
+
 import PostBodyStyles from '@/components/PostBodyStyles'
 import Toolbar from './Toolbar'
-import FormatBoldIcon from '@/components/Icons/FormatBoldIcon'
-import FormatItalicIcon from '@/components/Icons/FormatItalicIcon'
-import FormatUnderlinedIcon from '@/components/Icons/FormatUnderlinedIcon'
-import FormatTitleIcon from '@/components/Icons/FormatTitleIcon'
-import FormatLinkIcon from '@/components/Icons/FormatLinkIcon'
-import FormatQuoteIcon from '@/components/Icons/FormatQuoteIcon'
-import FormatListNumberedIcon from '@/components/Icons/FormatListNumberedIcon'
-import FormatListBulletedIcon from '@/components/Icons/FormatListBulletedIcon'
-import { ButtonType, withLinks, toggleMark, toogleByType, isTypeActive } from './helpers'
-import { useTranslation } from '@/config/i18n'
+import RenderElement from './RenderElement'
+import RenderLeaf from './RenderLeaf'
+import { withLinks, toggleMark, options } from './helpers'
```

Comment:
Thanks a lot for refactoring and cleaning up this file! It was definitely getting a bit crazy 😄

Reply:
😇